opm-bundle/manifests/lso-metrics.yaml

```diff
@@ -0,0 +1,57 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: local-storage-monitor
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: local-storage-monitor
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: local-storage-monitor
+subjects:
+- kind: ServiceAccount
+  name: local-storage-admin
+  namespace: openshift-local-storage
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: local-storage-metrics
+  namespace: openshift-local-storage
+rules:
+ - apiGroups:
+   - ""
+   resources:
+    - services
+    - endpoints
+    - pods
+   verbs:
+    - get
+    - list
+    - watch
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: local-storage-metrics
+  namespace: openshift-local-storage
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: local-storage-metrics
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: openshift-monitoring
+---
```

Comment:
Lets not make any direct changes to this directory.

Reply:
This is for the monitoring rbac rules.  Adding them the the bundle. I haven't see operators adding the monitoring related permissions in the CSV.

Reply:
which operators you are looking at? Please note - ART team tooling *only* looks at - https://github.com/openshift/local-storage-operator/tree/master/config/manifests directory as final version of manifests it needs to apply. `opm-bundle` directory is for our own hacking/testing purposes.

Reply:
Thanks for following up the discussion with ART and operator-sdk folks. So at very minimum- we need this rbac policy in `config/manifests` ?

Reply:
looks like it.